interface/portraits/portraits_tauren.gfx

```diff
@@ -12,8 +12,21 @@ spriteTypes = {
 		layer = { # GFX_TYPE:[d|p]INDEX:COLOR_LINK:DONT_REFRESH_IF_VALID:CULTURE_INDEX
```

Comment:
The horn and headgear overlay each other in bad way.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105850320-062d6380-5ffb-11eb-86d2-6593f06cf89a.png)

</details>